e2e-tests/development-runtime/src/pages/collection-routing/{FakeData.uuid}.js

```diff
@@ -15,7 +15,7 @@ export default function FakeData({ data: { fake }, pageContext: { uuid } }) {
 }
 
 export const blogPostQuery = graphql`
-  query GetFakeDataByUUIDCollection($id: String!) {
+  query($id: String!) {
```

Comment:
Since I copy/pasted the query multiple times it's easier to let Gatsby handle the generation of the name